op-devstack/presets/orchestrator.go

```diff
@@ -100,15 +99,21 @@ func initOrchestrator(ctx context.Context, p devtest.P, opt stack.CommonOption)
 	if lockedOrchestrator.Value != nil {
 		return
 	}
-	p.Logger().WithContext(ctx).Info("initializing orchestrator", "backend", globalBackend)
-	switch globalBackend {
-	case SysGo:
+
+	backend := "sysgo"
```

Comment:
It would be nice to keep these as constants rather than embedding the strings directly.